src/libraries/System.Runtime/tests/System/UIntPtrTests.cs

```diff
@@ -462,5 +462,89 @@ public static IEnumerable<object[]> Parse_ValidWithOffsetCount_TestData()
             yield return new object[] { "12", 1, 1, NumberStyles.HexNumber, null, (UIntPtr)0x2 };
             yield return new object[] { "$1,000", 1, 3, NumberStyles.Currency, new NumberFormatInfo() { CurrencySymbol = "$" }, (UIntPtr)10 };
         }
+
+        [Theory]
+        [MemberData(nameof(Parse_ValidWithOffsetCount_TestData))]
+        public static void Parse_Span_Valid(string value, int offset, int count, NumberStyles style, IFormatProvider provider, UIntPtr expected)
```

Comment:
It looks like the failures for this one are because `Parse_ValidWithOffsetCount_TestData` is casting `offset` and `count` to `IntPtr`, which isn't correct given they are used as part of `AsSpan`.

Looks like this is a pre-existing failure and the test data wasn't being used until this PR

Reply:
So I just remove these casts?

Reply:
Right, you would remove the cast from the second and third argument, but not from the last. https://github.com/dotnet/runtime/pull/47595/files/b2c52b2a7dc13feae2001a55721e39bba2dccf82#diff-0b1f3c3826a374c251afa76e3234f9c10aeda984ded5d726262545c955014697R847, for example, would become: `yield return new object[] { "2147483647", 1, 9, NumberStyles.None, null, (IntPtr)147483647 };` so they types match the expected